### PR TITLE
Web Min Version Test

### LINKED ISSUE #1464: Compile web version in `min_version_test` CI
### Description

Right now we only compile Android and iOS in the `min_version_test.yml` CI, but Flutter web uses different tooling, better to compile the Web version as well.

### PR DESCRIPTION
**Pull request by denrase**
#skip-changelog

## :scroll: Description

Build min version web target.

## :bulb: Motivation and Context

Closes #1464

## :green_heart: How did you test it?

Run CI

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] No new PII added or SDK only sends newly added PII if `sendDefaultPii` is enabled
- [ ] I updated the docs if needed
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: .github/workflows/min_version_test.yml [hunk 1]
```diff
@@ -56,3 +56,20 @@ jobs:
...
+
+      - uses: subosito/flutter-action@48cafc24713cca54bbe03cdc3a423187d413aafa # pin@v2.10.0
+        with:
+          flutter-version: '3.0.0'
+
+      - name: Build web
+        run: |
+          cd min_version_test
+          flutter pub get
+          flutter build web
```
Comment (marandaneto) [inline .github/workflows/min_version_test.yml]: Lets pass the source maps parameter as well

File: min_version_test/lib/main.dart [hunk 2]
```diff
@@ -112,9 +112,13 @@ class _MyHomePageState extends State<MyHomePage> {
...
 
       try {
+
+        // TODO Handle web file error
+
         final file = File('response.txt');
         final response = await dio.get<String>('https://flutter.dev/');
-        await file.writeAsString(response.data ?? 'no response');
+        print(response);
+        // await file.writeAsString(response.data ?? 'no response');
```
Comment (marandaneto) [inline min_version_test/lib/main.dart]: Why did we change this? what is the error to be handled?

Comment (denrase) [reply min_version_test/lib/main.dart]: Can't build the web target, as we dont have file i/o. Need to do this conditionally.

Comment (marandaneto) [reply min_version_test/lib/main.dart]: Can we fix the sample instead using conditional imports?

Example:
```dart
import '_io_environment_variables.dart'
    if (dart.library.html) '_web_environment_variables.dart'
```

Where the code does the same but one of them is NoOp, we can use this approach to conditionally import what we need.

Comment (denrase) [reply min_version_test/lib/main.dart]: Split the transaction sample in two files. One for web, the other for iOS/Android.

**Review by marandaneto — APPROVED** [review]

**Pull request merged**